scripts/qiita

```diff
@@ -437,7 +438,14 @@ def start(port, master):
             raise
 
     click.echo("Qiita started on port %d" % port)
-    tornado.ioloop.IOLoop.instance().start()
+    ioloop = tornado.ioloop.IOLoop.instance()
+
+    def callback_function():
+        for s in qdb.software.Software.iter_active():
+            s.register_commands()
+
+    ioloop.add_timeout(time() + 0.5, callback_function)
```

Comment:
Maybe more convenient to use tornado's time? http://www.tornadoweb.org/en/stable/ioloop.html#tornado.ioloop.IOLoop.time

Not suer if it is available in the version of tornado we are using.